examples/deployment/kubernetes/create.sh

```diff
@@ -23,7 +23,7 @@ if ! jq --help > /dev/null; then
 fi
 
 echo "Creating new Trillian deployment"
-echo "  Project name: ${PROJECT_NAME}"
+echo "  Project name: ${PROJECT_ID}"
 echo "  Cluster name: ${CLUSTER_NAME}"
 echo "  Region:       ${REGION}"
 echo "  Node Locs:    ${NODE_LOCATIONS}"
```

Comment:
Add `${RUN_MAP} to the summary?

Reply:
I figured that was more about the deploy than the creation?

Reply:
Yeah maybe. Might be worth adding it somewhere that there's no map being set up but not a big deal.